test/spec/bidmanager_spec.js

```diff
@@ -264,7 +264,7 @@ var bidmanager = require('../../src/bidmanager');
                         }
                 };
 
-                var expected = {"hb_bidder":  bidderCode, "hb_adid": adId,"hb_pb": 15.0 };
```

Comment:
Maybe we could also add some tests to flesh out the edge cases that use `alwaysUseBid`.  I don't see any places where we are testing that behavior.